packages/deployer/src/build/plugins/node-modules-extension-resolver.test.ts

```
@@ -1,16 +1,19 @@
 import { describe, it, expect, vi, beforeEach } from 'vitest';
 import type { Plugin, PluginContext } from 'rollup';
+import { getPackageInfo } from 'local-pkg';
+import { readFile } from 'node:fs/promises';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
 
-const mockResolveFrom = vi.fn();
-const mockReadFileSync = vi.fn();
+const mockNodeResolveHandler = vi.fn();
 const mockNodeResolveHandler = vi.fn();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'mockNodeResolveHandler'. Consider to delete it or rename it.

'mockNodeResolveHandler' is defined here:


(lint/suspicious/noRedeclare)